admin-app/src/App.css

```diff
@@ -0,0 +1,42 @@
+.App {
```

Comment:
the way to style react-admin apps is via the sx prop, not a global CSS.

https://marmelab.com/blog/2022/02/27/mui-v5-rocks.html

Reply:
Ok, it's done in another branch, i will rebase on it.